apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-body.tsx

```diff
@@ -29,7 +36,7 @@ export const InAppBody = () => {
                 id={field.name}
                 extensions={[
                   liquid({
-                    variables: [{ type: 'variable', label: 'asdf' }],
+                    variables: step ? parseJSONSchemaToLiquidVariables(step.variables as any) : [],
```

Comment:
Why is the `as any` needed?

If it's really needed, I suggest we coerce to `as JsonSchema`

Reply:
It shouldn't be needed.

Reply:
fixed